### DISCUSSION TITLE
Pieces Support Chat Built into the App 👀

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#184] [Ideas]
Something we're thinking about, what does the community think?

**Comment by Raven32768** [comment]
One would think any AI integrated App with chat, would have provided the AI with info to assist the user. It's like they built the app and forgot about both the ai and the user in that regard. This is a great app, don't get me wrong, but when the ai genius that is writing code for you doesn't even know what app he is a part of, my goodness.
